From f19160ef1c448f23a8e64ee3b2af1a13afb62f38 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 31 Aug 2016 04:55:20 -0700
Subject: [PATCH] arm64: dts: rk3399-rev3: modify the cpu's opp table

The opp table can cover the chips whose leakage is between 30mA and 60mA.

Change-Id: I50be3923eb6016cba6309380006ce902d22fe123
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
index 9a58f9e1ce03..475837e07cb7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3.dtsi
@@ -83,7 +83,7 @@
 	};
 	opp@1008000000 {
 		opp-hz = /bits/ 64 <1008000000>;
-		opp-microvolt = <875000>;
+		opp-microvolt = <850000>;
 	};
 	opp@1200000000 {
 		opp-hz = /bits/ 64 <1200000000>;
@@ -95,7 +95,7 @@
 	};
 	opp@1512000000 {
 		opp-hz = /bits/ 64 <1512000000>;
-		opp-microvolt = <1075000>;
+		opp-microvolt = <1100000>;
 	};
 };
 
@@ -115,27 +115,27 @@
 	};
 	opp@1008000000 {
 		opp-hz = /bits/ 64 <1008000000>;
-		opp-microvolt = <875000>;
+		opp-microvolt = <850000>;
 	};
 	opp@1200000000 {
 		opp-hz = /bits/ 64 <1200000000>;
-		opp-microvolt = <950000>;
+		opp-microvolt = <900000>;
 	};
 	opp@1416000000 {
 		opp-hz = /bits/ 64 <1416000000>;
-		opp-microvolt = <1025000>;
+		opp-microvolt = <1000000>;
 	};
 	opp@1608000000 {
 		opp-hz = /bits/ 64 <1608000000>;
-		opp-microvolt = <1100000>;
+		opp-microvolt = <1050000>;
 	};
 	opp@1800000000 {
 		opp-hz = /bits/ 64 <1800000000>;
-		opp-microvolt = <1175000>;
+		opp-microvolt = <1150000>;
 	};
 	opp@1992000000 {
 		opp-hz = /bits/ 64 <1992000000>;
-		opp-microvolt = <1250000>;
+		opp-microvolt = <1225000>;
 	};
 };
 
-- 
2.35.3

